README.md

```diff
@@ -1876,6 +1876,7 @@ Update Time, five active automations, webhooks.
 ## Other Free Resources
 
   * [Buff.tools](https://buff.tools/) - An all-in-one digital toolbox featuring Web, SEO, AI, Domain Management, unit conversion, calculators, and Image Manipulation Tools.
+  * [Htmlrev.com](https://htmlrev.com/) - Get 1000+ Selected free HTML, Bootstrap, Tailwind templates for websites, landing pages, coming soon, blogs, portfolios, ecommerce, dashboards and more.
```

Comment:
This is just an index to other sites, on its own not exactly a servive?

Reply:
The service **HTMLrev** provides is manually selecting good quality templates that: 

1. Are free for both personal and commercial projects
2. Are based on updated tech 
3. Have live preview links. 

Saving developers time and headaches from searching through low quality items with unusable licenses.

It also gives new templates a chance to be showcased as they will never get a chance to appear in Google search results